client/main.js

```diff
@@ -7,36 +7,32 @@ import format from 'date-fns/format';
 import hrLocale from 'date-fns/locale/hr'
 
 function Event({ event }) {
+  const { category, title, date, time, location, description, image, link } = event;
+  const Description = desc => ({ __html: desc });
   const EventLink = ({ link }) => link ? <a href={ link.url }>{ link.label }</a> : null;
   const EventTime = ({ time }) => time ? <span>@{ time } h</span> : null;
```

Comment:
One could argue that those two are now pretty generic and should be named `Link` and `Time` respectfully? 🤔 
Also they should probably end up being pulled outside from `Event` component...